QUEST_JOBSTEP.tsv

```diff
@@ -2331,20 +2331,20 @@ QUEST_JOBSTEP_20150323_002330	Reject since it seems too dangerous
 QUEST_JOBSTEP_20150323_002331	Ask him what the gift is
 QUEST_JOBSTEP_20150323_002332	Complete the mission of the Elementalist[Elementalist Advancement]
 QUEST_JOBSTEP_20150323_002333	%s defeated
-QUEST_JOBSTEP_20150323_002334	To prove your determination to be stronger, please collect Glizardon's Leather from the 1st basement of Tenet Cathedral, Galok's Canine from the 1st floor, and Black Powders from Egnome at the 2nd floor.
+QUEST_JOBSTEP_20150323_002334	To prove your determination to be stronger, please collect Glizardon's Leather from the 1st basement of Tenet Chapel, Galok's Canine from the 1st floor, and Black Powders from Egnome at the 2nd floor.
```

Comment:
1st basement of Tenet Chapel -> Tenet Chapel B1
1st floor -> 1F
2nd floor -> 2F